File/Type/SvgFile.php

```diff
@@ -15,7 +15,9 @@ class SvgFile extends FileType
      */
     public function processForDev (Asset $asset, string $filePath, string $fileContent) : string
     {
-        $header = $this->generateGenericFileHeader($asset, $filePath, '<!--', '-->');
-        return $header . $fileContent;
+        // the comment must be at the bottom, because if the SVG has a <?xml .. tag, it needs to be the
+        // very first thing in the file and this debug info would be above it.
+        $footer = $this->generateGenericFileDebugInfo($asset, $filePath, '<!--', '-->');
```

Comment:
This method is missing, isn't it? At least it's not part of this PR (yet) and I can't find it in the current repo.

Reply:
The method is part of this PR, take a look in the trait.

Reply:
Oh wow, I must be blind today.